docs/book/src/Good Practices

```diff
@@ -0,0 +1,24 @@
+# Good Practices
+
+## What is "Reconciliation" in Operators?
+
+The reconciliation is a continuos  loop that performs necessary actions on the current state to ensure that the Custom Resource reaches the desired state as specified by the user.
+
+## Why should reconciliations be idempotent?
```

Comment:
IMO we need to define here solutions and not reconciliations

Reply:
okay, what solutions you would like to see. will you gide me to reference docs? or elaborate